src/qt/applicationbase/undomanager.cpp

```diff
@@ -97,13 +95,6 @@ class AutoSaver {
                 }
 
                 if (str) {
-                    int numRestoreFiles = 100000;
```

Comment:
The new version will need to restart inviwo to have this take effect... :( 
Better to pass a lambda if you don't what to couple to the editorsettings

Reply:
No change from before

Reply:
Yes this is a change from before since it was pull on every save before

Reply:
fixed